src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -658,6 +658,19 @@ void xocl_mm_update_usage_stat(struct xocl_drm *drm_p, u32 ddr,
 	drm_p->mm_usage_stat[ddr]->bo_count += count;
 }
 
+int xocl_bo_update_usage_stat(struct xocl_drm *drm_p, int bo_idx,
```

Comment:
from the places where this function is called, you always convert bo flags to index, then pass it into this function here. why you can't use bo flags directly as input and find out the index internally? or do you expect the caller of this function may not know the bo flags in some cases?

Reply:
bo types are defined in xocl_bo.h and may not be good to inclide this file in xocl_drm

Reply:
why you can't include it? it is just a header...

Reply:
Moved implementation to xocl.bo.c This is best